src/main/java/mate/jdbc/manufacturerdao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,108 @@
+package mate.jdbc.manufacturerdao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturersStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturersStatement
+                        .executeQuery("SELECT * FROM manufacturers where is_deleted = false");
+            while (resultSet.next()) {
+                String manufacturerName = resultSet.getString("name");
+                String manufacturerCountry = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer =
+                        new Manufacturer(id, manufacturerName, manufacturerCountry);
+                manufacturers.add(manufacturer);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", throwables);
+        }
```

Comment:
```suggestion
            while (resultSet.next()) {
                manufacturers.add(parseManufacturerFromResultSet(resultSet));
            }
        } catch (SQLException e) {
            throw new DataProcessingException("Can't get all manufacturers from DB", e);
        }
```
move manufacturer parsing logic in get and getAll methods to separate private method